components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -4685,6 +4825,104 @@ public int getApplicationsCount(int tenantId, String searchOwner, String searchA
         return consumerKeys.toArray(new String[consumerKeys.size()]);
     }
 
+    public List<Application> getApplicationsByOrgId(String orgId) {
```

Comment:
Shall we not use orgId or organizationId. Use `organization` . This was the convention since this is used in both onprem and choreo